### ISSUE TITLE
Form Fields are Fillable, even when hidden

### ISSUE THREAD

**Issue by andrewsgarver** [phegman/v-show-slide#5]
When wrapping form fields in a v-show-slide, the user can tab into hidden form fields and edit them

**Comment by phegman** [comment]
This should be fixed in the latest release (v1.7.0). I also added an Accessibility section to the documentation.
